NapDatabaseExport/MSSQLServerProvider.cs

```diff
@@ -95,7 +112,7 @@ private SqlCommand GetCommand (string commandText, IEnumerable<DbParam> paramete
 
         public override bool TryConnect ()
         {
-            if (string.IsNullOrWhiteSpace (Server) || string.IsNullOrWhiteSpace (User))
+            if (string.IsNullOrWhiteSpace (Server))
```

Comment:
Better check the user if it is empty and required instead of not checking at all for it

Reply:
The point is that *empty* user signifies "Use integrated security i.e. current logged on user security context without bothering them w/ entering password"

Reply:
OK